app/models/freeform.rb

```diff
@@ -2,6 +2,10 @@ class Freeform < Tag
 
   NAME = ArchiveConfig.FREEFORM_CATEGORY_NAME
 
+  def self.label_name
+    "Additional Tags"
```

Comment:
Maybe we should have used `ArchiveConfig.FREEFORM_CATEGORY_NAME` for this. Same with `label_name` of the other tag types.